### DISCUSSION TITLE
Problemas de inicio de sesión

### DISCUSSION THREAD

**Discussion by AlexAlz** [LaunchX-InnovaccionVirtual/CursoIntroPython#340] [Q&A]
Buenas tardes compañeros, gracias a mí falta de memoria de corto plazo, olvidé mí contraseña del correo de innovación y no he podido entregar las katas faltantes.
Me he comunicado ya con contacto innovación, pero no obtengo respuesta ¿Saben a quién más podría contactar?

**Comment by RIvanCF** [comment]
En el perfil de instagram de Innovaccion solicitas que te repongan tu contraseña

**Reply by AlexAlz** [reply]
Muchas gracias! <3
